docs/sources/network/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Network metrics
+menuTitle: Network
+description: Configuring Beyla to observe point-to-point network metrics.
+weight: 1
+keywords:
+  - Beyla
+  - eBPF
+  - Network
+---
+
+{{% admonition type="warning" %}}
+Network metrics is an [experimental](/docs/release-life-cycle/) under development feature, expect breaking changes.
+{{% /admonition %}}
+
+# Network metrics
+
+Grafana Beyla can be configured to provide network metrics between different endpoints. For example, between physical nodes, containers, Kubernetes pods, services, etc.
+
+{{% admonition type="note" %}}
+Prometheus exporting for network metrics is not currently supported.
+{{% /admonition %}}
+
+## Get started
+
+To get started using Beyla networking metrics, consult the [quickstart setup documentation]({{< relref "./quickstart" >}}), and for advanced configuration, consult the [configuration documentation]({{< relref "./config" >}}).
+
+## Metric attributes
+
+Network metrics provides a single **OpenTelemetry** metric `beyla.network.flow.bytes`, a counter of the total bytes sent of network flows observed by the eBPF probe since its launch, with the following attributes:
+
+- `beyla.ip`: the local IP address of the Beyla instance that emitted the metric
```

Comment:
Are the attribute pairs mutually exclusive or can they both occur? Could we separate them to their own lines, even better can we use a table?

Reply:
They can Both occur. I will use a table, thanks for the suggestion!